ingestion/src/metadata/ingestion/source/database/dbt_service.py

```diff
@@ -139,14 +131,9 @@ class DbtServiceSource(TopologyRunnerMixin, Source, ABC):
     context = create_source_context(topology)
 
     def get_dbt_files(self) -> DbtFiles:
-        dbt_details = get_dbt_details(
+        dbt_files = get_dbt_details(
             self.source_config.dbtConfigSource  # pylint: disable=no-member
         )
-        dbt_files = DbtFiles(
-            dbt_catalog=dbt_details[0],
-            dbt_manifest=dbt_details[1],
-            dbt_run_results=dbt_details[2],
-        )
```

Comment:
instead of expecting a tuple here directly returning the `DbtFiles` object to avoid any index out of bound error

Reply:
better, thank you